client/checkout/classic/upe-deferred-intent-creation/test/payment-processing.test.js

```diff
@@ -377,13 +376,46 @@ describe( 'Payment processing', () => {
 
 		const checkoutForm = {
 			submit: jest.fn(),
-			addClass: jest.fn( () => {
-				return {
-					block: jest.fn(),
-				};
-			} ),
-			removeClass: jest.fn(),
-			unblock: jest.fn(),
```

Comment:
I noticed that `unblock` is called after `removeClass` (similar to `addClass().block()`). So I fixed it on all the tests.